src/tasks/level10.ts

```diff
@@ -31,14 +31,29 @@ export const GiantQuest: Quest = {
       after: ["Grow Beanstalk"],
       completed: () => have($item`S.O.C.K.`),
       do: $location`The Penultimate Fantasy Airship`,
-      choices: { 178: 2, 182: 1 },
+      choices: {
+        178: 2,
+        182: () => {
+          // eslint-disable-next-line libram/verify-constants
+          if (haveEquipped($item`bat wings`)) return 6;
```

Comment:
I think we might need a choice adventure script for this? Since I think KoLMafia will abort if we hit this but we have not cleared enough delay to actually use bat wings.

Reply:
Ah I see, right